YogaKit/YGLayout.m

```diff
@@ -318,6 +312,14 @@ static void YGAttachNodesFromViewHierachy(UIView *const view)
   if (yoga.isLeaf) {
     YGRemoveAllChildren(node);
     YGNodeSetMeasureFunc(node, YGMeasureView);
+
+    YGSize oldSize = yoga.cachedSize;
+    YGSize newSize = YGMeasureView(node, 0, YGMeasureModeUndefined, 0, YGMeasureModeUndefined);
```

Comment:
@hartbit Sorry for the delay in responding. I would rather not implement these kinds of flags for now. Is it easy enough to implement outside of Yoga within applications which would like to use this? I think we should continue to think about this irregardless of whether we end up merging this PR, I know @dshahidehpour is very interested in solving this as well.